test/correctness/fuzz_simplify.cpp

```diff
@@ -319,11 +339,11 @@ int main(int argc, char **argv) {
     rng.seed(fuzz_seed);
     std::cout << "Simplify fuzz test seed: " << fuzz_seed << "\n";
 
-    int max_fuzz_vector_width = 4;
+    int max_fuzz_vector_width = 8;
 
     for (int i = 0; i < fuzz_type_count; i++) {
         Type T = fuzz_types[i];
-        for (int w = 1; w < max_fuzz_vector_width; w *= 2) {
+        for (int w = 1; w <= max_fuzz_vector_width; w *= 2) {
```

Comment:
I think the test as-is will only test power-of-2 width vectors, but we do support other vectorization widths and it might be good to test those with the new ramp-of-ramps logic. 6 would be a good one (to get 3 x 2), so maybe this should just be a loop like: `for (int w : {1, 2, 3, 4, 6, 8})`. Or maybe we should just use random vector widths with a large number of trials.

IIRC the run time of this test was an issue, which is why I'm guessing we tried to minimize the "branching factor" here.

Reply:
I've changed it to randomly pick a vector width (and type) and included 3 and 6 in the list. Also, increased number of trials.